index.html

```diff
@@ -85,6 +85,37 @@ <h2>Start your spec!</h2>
       The <dfn>bar</dfn> attribute.
     </p>
   </section>
+  <section id="RTCAudioReceiverStats-dict*">
+    <h3>
+      <dfn>RTCAudioReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCAudioReceiverStats {
+      unsigned long long jitterBufferFlushes;
+      unsigned long long delayedPacketOutageSamples;
+};</pre>
+    <dl data-link-for="RTCAudioReceiverStats" data-dfn-for="RTCAudioReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>jitterBufferFlushes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Counts the total number of times the jitter buffer
+          reaches its maximum capacity and gets flushed.
```

Comment:
Just because you are full does not mean that you will flush it?
Just because you flush it does not mean that that the jitter buffer was full?

How about counting the number of samples that are thrown away due to flushing instead?